docs/esignet-signup-openapi.yaml

```diff
@@ -646,6 +666,585 @@ paths:
+                    properties:
+                      terms&Conditions:
+                        type: object
+                        description: |
+                          List of terms and conditions specific to identity verifier. Details of the identity verifier is fetched from the config server. File is identified with the identity verifier ID.
+                          File name format: signup-idv_%s.json
+                          %s is replaced with Identity verifier ID.
+                        x-stoplight:
+                          id: 04wwvclq7tmh8
+                        additionalProperties:
+                          x-stoplight:
+                            id: 2ibluuduvuut6
+                          type: string
+                      errors:
+                        type: object
+                        x-stoplight:
+                          id: rs17urkck31b6
+                        additionalProperties: {}
+                      messages:
+                        type: object
+                        x-stoplight:
+                          id: xrtxlq8e76cdz
+                        additionalProperties: {}
+                  errors:
+                    type: array
+                    items:
+                      type: object
+                      properties:
+                        errorCode:
+                          type: string
+                          enum:
+                            - invalid_transaction
+                            - invalid_identity_verifier_id
+                        errorMessage:
+                          type: string
+          headers: {}
+      servers:
+        - url: 'https://signup.collab.mosip.net/v1/signup'
+      x-stoplight:
+        id: 8k3y829yk0k5g
+    parameters:
+      - schema:
+          type: string
+        name: id
+        in: path
+        required: true
+        description: Identity verifier identifier
+  /identity-verification/slot:
+    post:
+      tags:
+        - UI
+      summary: Fetch Slot ID endpoint
+      description: |
+        Endpoint to get the slot ID to begin the video identity verification process.
+
+        1. Validates the cookie with valid IDV transaction ID.
+        2. Validate the Identity verifier ID and should be `active`.
+        3. Validate if the consent is provided, if consent is not provided reject the request.
+        4. Check if the disabilityType is set with valid value, null is allowed.
+
+        Video identity verification process is handled using websocket connection. 
+        As the server limits the number of websocket connection, every request to get the slot ID checks the current ongoing websocket connections.
+        slot ID is returned only if the current count is less than the allowed count. Otherwise, `slot_not_available` error is returned.
+
+        **Note:** If this request is invoked after getting a valid slot ID, Server returns back the same slot ID until the transaction timeout.
```

Comment:
I think these statements are not true according to the current code. Can you please check and update them?